website/docs/components/card/card-code.md

```diff
@@ -22,15 +22,17 @@ tabs: Design('card'), A11y('card-a11y'), API('card-api'), Example('card-code'),
 
 :::
 
-<!-- ## Card layout for tables
+## Card layout for tables
```

Comment:
What do you think about adding a link to this example somewhere on the DataTable examples page?